Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -438,6 +426,11 @@ void registerWebKitGStreamerElements()
             if (auto vaapiPlugin = adoptGRef(gst_registry_find_plugin(registry, "vaapi")))
                 gst_registry_remove_plugin(registry, vaapiPlugin.get());
         }
+
+        // Make sure the quirks are created as early as possible.
+        auto& quirksManager = GStreamerQuirksManager::singleton();
+        UNUSED_VARIABLE(quirksManager);
```

Comment:
Any reason to not go with `maybe_unused`?

```suggestion
        [[maybe_unused]] auto& quirksManager = GStreamerQuirksManager::singleton();
```

Reply:
No special reason :)